source/math/FloatBox2D.ooc

```diff
@@ -187,4 +187,12 @@ FloatBox2D: cover {
 		}
 		This new(xMinimum, yMinimum, xMaximum - xMinimum, yMaximum - yMinimum)
 	}
+	linearInterpolation: static func (a, b: This, ratio: Float) -> This {
+		FloatBox2D new(
+			Float linearInterpolation(a left, b left, ratio),
+			Float linearInterpolation(a top, b top, ratio),
+			Float linearInterpolation(a width, b width, ratio),
+			Float linearInterpolation(a height, b height, ratio)
+		)
```

Comment:
When jumping more than one step to the left on the same line, it looks broken and draws attention.